NewsBook/Application/Commands/FavouriteNews/FavouriteNewsCommands.cs

```diff
@@ -0,0 +1,18 @@
+﻿using MediatR;
+using NewsBook.Mediator.Response;
+
+namespace NewsBook.Mediator.Commands.FavouriteNewsCommands
+{
+    public class MarkFavouriteNews : IRequest<NewsReadDTO>
```

Comment:
Follow consistent naming conventions across the project.

The classes in the Commands namespace should be prefixed with the "Command" word.